app/views/jobs/getjob.html.erb

```diff
@@ -31,10 +31,10 @@
 </tr></table>
 
 <% if @cud.user.administrator?  then %>
-<li><kbd>notifyURL</kbd>: <kbd><%= @job[:rjob]["notifyURL"] %> </kbd>
+<li><kbd>notifyURL</kbd>: <%= @job[:rjob]["notifyURL"] %>
```

Comment:
@dlbucci Because notifyURL is kind of a "reserved word", while I changed the other lines to normal English whenever possible...